Some extensions don't like parallel sphinx-build
--- mongo-c-driver-1.16.2/build/cmake/SphinxBuild.cmake.orig	2020-02-25 06:38:41.000000000 +0100
+++ mongo-c-driver-1.16.2/build/cmake/SphinxBuild.cmake	2020-05-03 16:05:03.406177913 +0200
@@ -40,7 +40,7 @@
       ${CMAKE_COMMAND} -E env
       "PYTHONDONTWRITEBYTECODE=1"
       ${SPHINX_EXECUTABLE}
-         -j ${NPROCS} -qEW -b html
+         -qEW -b html
          -c "${CMAKE_CURRENT_SOURCE_DIR}"
          "${CMAKE_CURRENT_SOURCE_DIR}"
          "${SPHINX_HTML_DIR}"
@@ -133,7 +133,7 @@
       ${CMAKE_COMMAND} -E env
       "PYTHONDONTWRITEBYTECODE=1"
       ${SPHINX_EXECUTABLE}
-         -j ${NPROCS} -qEW -b man
+         -qEW -b man
          -c "${CMAKE_CURRENT_SOURCE_DIR}"
          "${CMAKE_CURRENT_SOURCE_DIR}"
          "${SPHINX_MAN_DIR}"
